scripts/publish_packages_to_local_registry.py

```diff
@@ -3,7 +3,6 @@
 # ------------------------------------------------------------------------------
 #
 #   Copyright 2021-2022 Valory AG
-#   Copyright 2018-2019 Fetch.AI Limited
```

Comment:
Revert please

Reply:
This script was created by me when I implemented the IPFS registry, This has nothing to do with Fetch.ai